reg/drone/phy/kinematics/trans/Velocity1CovTs.0.1.uavcan

```diff
@@ -0,0 +1,9 @@
+# Linear velocity with timestamp and covariance.
+# Observe that this is a structural subtype of uavcan.si.sample.velocity.Scalar.1.0.
+# For a non-timestamped estimate without covariance use the raw SI type directly.
+
+uavcan.si.sample.velocity.Scalar.1.0 velocity
+
+float16 error_variance  # [(meter/second)^2]
```

Comment:
Same thing here about the usage of Cov.

Reply:
This is not right. if you are talking about variance, then the variance needs to be in X, Y and Z.

Reply:
It is a one-dimensional state.

Reply:
which one? X, Y or Z?